simulation/SyntheSimJava/src/main/java/com/autodesk/synthesis/ctre/TalonFXConfigurator.java

```diff
@@ -0,0 +1,26 @@
+package com.autodesk.synthesis.ctre;
+
+import com.ctre.phoenix6.hardware.DeviceIdentifier;
+import com.ctre.phoenix6.configs.TorqueCurrentConfigs;
+import com.ctre.phoenix6.StatusCode;
+
+//import com.ctre.phoenix6.configs.TalonFXConfigurator
```

Comment:
Leaving this for the future?

Reply:
wdym?

Reply:
Wondering if this was also left for testing or not.

Reply:
it was not